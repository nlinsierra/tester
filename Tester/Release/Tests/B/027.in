TD 6S 2C JC AC 8S 2D 4D 3S QH TH KS JH
