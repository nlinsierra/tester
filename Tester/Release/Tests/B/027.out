39
3C 4C 5C 6C 7C 8C 9C TC QC KC 3D 5D 6D 7D 8D 9D JD QD KD AD 2H 3H 4H 5H 6H 7H 8H 9H KH AH 2S 4S 5S 7S 9S TS JS QS AS
